update services articles in v14 (#5987)
umbraco/UmbracoDocs: 14/umbraco-cms/reference/management/services/retrieving-content-types.md

// Check if there are children
bool hasChildren = _contentTypeService.HasChildren(1234);
```

# Retrieving content type container

## Getting a single content type container

> Content types can be added either at the root level, under another content type or under a content type container (container is a folder). The approach for getting a single container is similar to getting a single content type. This means that you can look up a container - either by its GUID:

Reviewer: ```suggestion
You can add content types in three different ways - at the root level, under another content type, or under a container (which is essentially a folder). To obtain a single container, the process is similar to obtaining a single content type. This means that you can search for a container either by its GUID:
```

> You can add content types in three different ways - at the root level, under another content type, or under a container (which is essentially a folder). To obtain a single container, the process is similar to obtaining a single content type. This means that you can search for a container either by its GUID:

Author: ```suggestion
You can add content types in three different ways. At the root level, under another content type, or under a _container_ (which is a folder). To obtain a single container, the process is similar to obtaining a single content type. This means that you can search for a container either by its GUID:
```

> Content types can be added either at the root level, under another content type or under a content type container (or folders as they're called in the Umbraco backoffice). The approach for getting a single container is similar to getting a single content type, meaning that you can look up a container - either by its GUID:

Author: ```suggestion
Content types can be added either at the root level, under another content type or under a content type container (container is a folder). The approach for getting a single container is similar to getting a single content type. This means that you can look up a container - either by its GUID:
```
